### PR TITLE
FWK-670-nvme-blkid-issue

### PR DESCRIPTION
**Pull request by pploegaert**

### CODE REVIEW

File: ovs/lib/disk.py [hunk 1]
```diff
@@ -63,14 +63,20 @@ def sync_with_reality(storagerouter_guid=None):
             # Gather mount data
             mount_mapping = {}
             mount_data = client.run('mount')
+            blkids = dict()
+            for entry in client.run('blkid').splitlines():
+                dev_name, dev_details = entry.split(': ')
+                if 'UUID' in dev_details:
+                    blkids[str(dev_name)] = str(dev_details.split('"')[1])
+                continue
```
Comment (khenderick) [inline ovs/lib/disk.py]: This line doesn't add much

**Comment by khenderick** [comment]
The travis issue might be solved if you merge in master.

**Comment by pploegaert** [comment]
Test results: http://testrail.openvstorage.com/index.php?/plans/view/33144

**Comment by khenderick** [comment]
Moved `state_verification` label to the related ticket #670

**Pull request merged**